services/quizzes/src/components/SubmissionComponents/Checkbox.tsx

```diff
@@ -0,0 +1,88 @@
+import { css } from "@emotion/css"
+import styled from "@emotion/styled"
+import React from "react"
+
+import { ItemAnswerFeedback } from "../../pages/api/grade"
+import { baseTheme } from "../../shared-module/styles"
+
+import { QuizItemSubmissionComponentProps } from "."
+
+interface StyledProps {
+  correct: boolean
+}
+
+const Option = styled.div<StyledProps>`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  border: 3px solid
+    ${({ correct }) => (correct ? baseTheme.colors.green[400] : baseTheme.colors.red[400])};
+  border-radius: 3px;
+  margin-bottom: 10px;
+  padding: 5px;
+`
+
+const CheckBoxFeedback: React.FC<QuizItemSubmissionComponentProps> = ({
+  public_quiz_item,
+  user_quiz_item_answer,
+  quiz_item_feedback,
+}) => {
+  const correct = user_quiz_item_answer.intData === 1
```

Comment:
```suggestion
  const checked = user_quiz_item_answer.intData === 1
```
Everything is correct